### PR TITLE
Update README w/ info about 4.18 and ESM

### PR DESCRIPTION
**Pull request by tomwayson**
first step towards #254

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,16 +206,21 @@ See below for information on how to [override ArcGIS styles](#overriding-arcgis-
...
 
-There are a few ways to [integrate the ArcGIS API for JavaScript with other frameworks and their tools](https://developers.arcgis.com/javascript/latest/guide/using-frameworks/), but esri-loader is the most versatile since it works in applications that:
+esri-loader has been the most versatile way to integrate the ArcGIS API for JavaScript with other frameworks and their tools since it works in applications that:
 - are built with _any_ loader/bundler, such as [webpack](https://webpack.js.org/), [rollup.js](https://rollupjs.org/), or [Parcel](https://parceljs.org)
-- use [framework tools](https://developers.arcgis.com/javascript/latest/guide/using-frameworks/#framework-tools) that discourage or prevent you from manually editing the webpack configuration
+- use framework tools that discourage or prevent you from manually editing their configuration
 - use either version [4.x](https://developers.arcgis.com/javascript/) _or_ [3.x](https://developers.arcgis.com/javascript/3/) of the ArcGIS API for JavaScript
+- make very limited use of the ArcGIS API and don't want to incure the cost of including it in their build
```
Comment (odoe) [inline README.md:219]: _incure_ -> incur?

File: README.md [hunk 2]
```diff
@@ -204,16 +206,21 @@ See below for information on how to [override ArcGIS styles](#overriding-arcgis-
...
 
-Unfortunately, you can't simply `npm install` the ArcGIS API and then `import` 'esri' modules in a non-Dojo application. The only reliable way to load ArcGIS API for JavaScript modules is using Dojo's AMD loader. However, when using the ArcGIS API in an application built with another framework, you typically want to use the tools and conventions of that framework rather than the Dojo build system.
+<p id="why-is-this-needed">Unfortunately, prior to version 4.18, you couldn't simply `npm install` the ArcGIS API and `import` it's modules. The only reliable way to load ArcGIS API for JavaScript modules was using Dojo's AMD loader. Rather than using Dojo to build your applicaiton, esri-loader provides a way to dynamically load modules at runtime from a hosted build of the ArcGIS API (such as the <a href="https://developers.arcgis.com/javascript/latest/guide/get-api/#cdn">CDN</a>) into applications built using modern tools and framework conventions.</p>
```
Comment (odoe) [inline README.md:213]: I think it's still worth mentioning here that some users might want to use esri-loader to take advantage of the fast cached CDN for their apps. That could still be a consideration.

**Review by odoe — APPROVED** [review]
Nice. I just had a little comment about the CDN, but nice work!

File: README.md [hunk 3]
```diff
@@ -204,16 +206,21 @@ See below for information on how to [override ArcGIS styles](#overriding-arcgis-
...
+esri-loader has been the most versatile way to integrate the ArcGIS API for JavaScript with other frameworks and their tools since it works in applications that:
 - are built with _any_ loader/bundler, such as [webpack](https://webpack.js.org/), [rollup.js](https://rollupjs.org/), or [Parcel](https://parceljs.org)
-- use [framework tools](https://developers.arcgis.com/javascript/latest/guide/using-frameworks/#framework-tools) that discourage or prevent you from manually editing the webpack configuration
+- use framework tools that discourage or prevent you from manually editing their configuration
 - use either version [4.x](https://developers.arcgis.com/javascript/) _or_ [3.x](https://developers.arcgis.com/javascript/3/) of the ArcGIS API for JavaScript
+- make very limited use of the ArcGIS API and don't want to incure the cost of including it in their build
+
+Most developers will prefer the convenience of being able to `import` from `@arcgis/core` directly, especially if their applicaion makes extensive use of the ArcGIS API. However, if `@arcgis/core` doesn't work in your application for whatever reason, esri-loader probably will.
```
Comment (andygup) [inline README.md:221]: Minor typo: "if their applicaion" > application

File: README.md (hunk 2 above)
Comment (andygup) [reply README.md:213]: Minor typo: "build your applicaiton" > application

**Review by andygup — CHANGES_REQUESTED** [review]
Just some small typos, otherwise looks great!

**Comment by tomwayson** [comment]
appliacaitionianain is hard to spell

**Pull request merged**